samples/aspnetcore/blazor/FlightFinder/NuGet.config

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <packageSources>
+    <add key="blazor-dev" value="https://dotnet.myget.org/F/blazor-dev/api/v3/index.json" />
```

Comment:
This won't be needed once it's live right?

Reply:
No, it’s only needed until 0.2.0 is published to NuGet.